README.md

```diff
@@ -32,7 +35,7 @@ Search results are displayed using Solr server which is embedded into YaCy. All
 
 ## Requirements
 * [Angular-cli](https://github.com/angular/angular-cli#installation)
-* node --version >= 6
+* node --version >= 8.2.1
 * npm --version >= 3
```

Comment:
@praveenojha33 Would npm version v3 or related work with node v8.* ?

Reply:
Since Project works fine with node v6. Should I live it as it is.

Reply:
Keep the minimum working version in README. Since it works for v6.*, it should be kept as it is.

Reply:
Done